docs/adr/0008-scheduler-route-consumption-and-transport-ownership-boundary.md

```
@@ -2,12 +2,12 @@
 
 Status: Accepted
 
-**Date**: 2026-06-02
+**Date**: 2026-06-02; revised 2026-08-21
 **Context**: Process/server scheduler routing
 
 ### Decision
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)